.circleci/config.yml

```diff
@@ -36,6 +36,11 @@ workflows:
               only: /^v.*/
             branches:
               only: /.*/
+          post-steps:
+            - persist_to_workspace:
+                root: testapp/build/outputs
+                paths:
+                  - mapping/
```

Comment:
You also need to update the path in the `mapping` jobs: `mapping/staging/mapping.txt`, etc.

Reply:
nice catch @corycaywood , the PR only shows the change so it was not obvious it missed that